### PR TITLE
ENH: Added Least Square Fitting of Bezier Surfaces

### PR DESCRIPTION
**Pull request by dalbenzioG**
At this stage, when sliding the MarkupDistanceContour a model node of the uniform sampled contour points will be generated.

### CODE REVIEW

File: Liver/Liver.py [hunk 1]
```diff
@@ -41,519 +41,594 @@
...
 import LiverSegments
 
+
 #
 # Liver
 #
 
 class Liver(ScriptedLoadableModule):
-  """Uses ScriptedLoadableModule base class, available at:
+    """Uses ScriptedLoadableModule base class, available at:
```
Comment (olevs) [inline Liver/Liver.py:56]: Change the indentation back to 2 spaces for now

**Review by olevs — CHANGES_REQUESTED** [review]

**Comment by olevs** [comment]
@dalbenzioG: Can you fix the indentation in Liver.py so we can merge your pull request?

**Comment by dalbenzioG** [comment]
> @dalbenzioG: Can you fix the indentation in Liver.py so we can merge your pull request?
It shoud be fixed now. Sorry, my bad. My Pycharm IDE refactors back to 4 spaces indentation after I modified the Liver.py file.

**Review by olevs — APPROVED** [review]
A lot the the Python code could probably be converted to C++. Also some long functions (like runSurfacefromEFD) could be improved by splitting them into smaller functions.
Still, let's merge this now and possibly rewrite the code later

**Comment by olevs** [comment]
> > @dalbenzioG: Can you fix the indentation in Liver.py so we can merge your pull request?
> > It shoud be fixed now. Sorry, my bad. My Pycharm IDE refactors back to 4 spaces indentation after I modified the Liver.py file.

We can change to 4 spaces if we want in a separate pull request after all other pull requests have been merged

**Pull request merged**